speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -306,6 +352,13 @@ static void LoadConfigSettings(void* p_param)
     // Set the max time setting from the xml file
     m_maxTime = std::stoi(GfParmGetStr(p_param, PRM_MAX_TIME, GFMNU_ATTR_TEXT, NULL));
 
+    const char* filePath = GfParmGetStr(p_param, PRM_BLACKBOX, "name", NULL);
```

Comment:
Make a PRM_ATTR_NAME macro for the string literal "name" that you use to store and read from the config file (or maybe use a macro for "path" instead, seems more descriptive of what you're actually storing)